### PR TITLE
updated architechture

### PR DESCRIPTION
**Pull request by JonasZagst**
imporoved documentation for architecture

### CODE REVIEW

**Review by V1nzent — COMMENTED** [review]
Sounds good. But we should exactly describe on which OS YAAC can be used. So lets focus on a few operating systems because we have to test it on all listed operation systems.

**Comment by JonasZagst** [comment]
> Sounds good. But we should exactly describe on which OS YAAC can be used. So lets focus on a few operating systems because we have to test it on all listed operation systems.

To my knowledge it runs and is tested on every desktop OS MacOS, Linux and Windows with WSL2 as every system has at least one developper using it. The only thing is the GOCV requirement which does run on every system it just requires extensive setup which was the reason for creating the docker image.

**Review by V1nzent — COMMENTED** [review]
looks good now

File: Pages/3Design/3.1.0Architecture.md [hunk 1]
```diff
@@ -37,43 +35,42 @@ The ViewModel mediates between the Model and the View. It exposes data and comma
...
+After we tested different programming languages and their compatibility to different frontend tools in the exploration sprints, we decided to use the programming language Go and combine it with the toolkit Fyne. The reasons for that are described in the subdivision [0.0.0 Research Phase](../0Research/0.0.0ResearchPhase.md).
 
 ### 3.3 Libraries
 
 ### Go Standard Library
 
-The Go Standard Library is structured in a modular fashion with each package addressing specific functionalities in a minimalist way. This enables the code's code organization, reusability and extensibility. As YAAC's programming language is Go, this library is used for many functions in the code.
+The full list of libraries and frameworks used in the project can be viewed in the [go_mod file](https://github.com/DHBW-SE-2023/YAAC/blob/master/go.mod). The most important libraries however are:
 
-### Tesseract
```
Comment (EvaMatz) [inline Pages/3Design/3.1.0Architecture.md:70]: I'd add the tesseract paragraph to the Wiki- it is a library that we use in our project

**Review by EvaMatz — COMMENTED** [review]
I'd add the tesseract paragraph to the libraries chapter as its one of two main libraries we use in YAAC for imageproc.
Otherwise, the commit looks good

File: Pages/3Design/3.1.0Architecture.md (hunk 1 above)
Comment (JonasZagst) [reply Pages/3Design/3.1.0Architecture.md:70]: Thesseract is described in 3.3 at gosseract including the description you provided yesterday

Comment (EvaMatz) [reply Pages/3Design/3.1.0Architecture.md:70]: found it- sorry. Looks good then!

**Review by EvaMatz — APPROVED** [review]

**Pull request merged**